app.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/topics-controllers");
+
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.use((err, req, res, next) => {
+  console.log(err);
+  if (err.status) {
+    res.status(err.status).send({ msg: err.message });
+  }
+  next(err);
+});
+
+app.use((err, req, res, next) => {
+  res.status(500).send({ msg: "internal server error" });
```

Comment:
I would move your console.log into the last error handler, as this is essentially acting as an error debugger, should all your other errors not be ran